### PR TITLE
Add composer require checker

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 ## 1.0.0 under development
 
-- Initial release.
+- Enh: Add composer require checker into CI
```
Comment (samdark) [inline CHANGELOG.md:5]:
```suggestion
- Enh #69: Add composer require checker into CI (xepozz)
```

Comment (vjik) [reply CHANGELOG.md:5]: Need `@xepozz`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 ## 1.0.0 under development
 
-- Initial release.
+- Enh #69: Add composer require checker into CI (xepozz)
```
Comment (arogachev) [inline CHANGELOG.md:5]:
```suggestion
- Enh #69: Add composer require checker to CI (@xepozz)
```

**Review by arogachev — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**